projects/ui/src/lib/components/po-chart/po-chart-base.component.ts

```diff
@@ -170,6 +171,10 @@ export abstract class PoChartBaseComponent implements OnChanges {
   @Input('p-options') set options(value: PoChartOptions) {
     if (value instanceof Object && !(value instanceof Array)) {
       this._options = value;
+
+      if (this._options.hasOwnProperty('legend') && typeof this._options.legend === 'boolean') {
```

Comment:
Precisa fazer o calculo do `svgContainerSize` se legend for `true`?

Reply:
Precisa sim se houve o toggle, caso o usuário alternar entre false e true